server.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "https://static.modelcontextprotocol.io/schemas/2025-10-17/server.schema.json",
+  "name": "com.microsoft/azure-devops",
+  "description": "The MCP server for Azure DevOps, bringing the power of Azure DevOps directly to your agents.",
+  "version": "2.3.0",
+  "title": "Azure DevOps MCP",
```

Comment:
```suggestion
  "title": "Azure DevOps",
```

I think the current title hard coded into GitHub MCP Registry has no ` MCP` suffix
<img width="1390" height="501" alt="image" src="https://github.com/user-attachments/assets/ded118f0-27aa-4c6c-b8bb-a2072604341c" />